media-gfx/kuickshow/kuickshow-0.11.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="forceoptional"
+KFMIN=6.0.0
+QTMIN=6.0.0
+inherit ecm kde.org
+
+DESCRIPTION="Image viewer based on KDE Frameworks"
+HOMEPAGE="https://userbase.kde.org/KuickShow"
+SRC_URI="mirror://kde/unstable/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="6"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	>=dev-qt/qtbase-${QTMIN}:6
```

Comment:
It will likely need at least `gui` and `widgets` USE deps here.

Reply:
Those seem to be part of qtbase now. qtgui and qtwidgets are in my system only in version 5.15.16.

Reply:
Yes. But they're USE optional, so your rebuild needs to make sure they are enabled.

Reply:
Is this what you meant?
`>=dev-qt/qtbase-${QTMIN}:6[gui,widgets]`

Reply:
Exactly. That's what we call USE deps.

Reply:
Ok, I've already added that in the latest push, in case my guess was correct.